__test__/work-orders/[id]/choose-option.test.js

```diff
@@ -1,20 +1,20 @@
-import UpdateJobPage from 'src/pages/repairs/jobs/[id]/update-job'
+import ChooseWorkOrderOption from 'src/pages/work-orders/[id]/close'
```

Comment:
Seems like github has got this file rename confused with https://github.com/LBHackney-IT/repairs-hub-frontend/pull/167/files#diff-6905a79791a9124df3a62a132b20d970b030c6fd5be92862eea83adef251c268L1-R1